docs/source/contents/4_draft_mapping.rst

```diff
@@ -754,6 +815,12 @@ Decision
 
 Don't map!
 
+`Example record - roth:1595 <https://digital.lib.utk.edu/collections/islandora/object/roth%3A1595/datastream/MODS/view>`
+
+.. code-block:: xml
+
+    </abstract>
```

Comment:
I'm thinking we want `<abstract/>` here to indicate the empty element.

Reply:
Yes, let me know if you think any changes are necessary. I'm not certain that we've been consistent throughout with showing empty elements like this.